packages/react/src/components/Cta/__stories__/Cta.stories.js

```diff
@@ -0,0 +1,127 @@
+import { select, withKnobs } from '@storybook/addon-knobs';
+import CTA from '../Cta';
+import React from 'react';
+import readme from '../README.md';
+import { storiesOf } from '@storybook/react';
+
+storiesOf('Components|CTA', module)
+  .addDecorator(withKnobs)
+  .addParameters({
+    readme: {
+      sidebar: readme,
+    },
+  })
+  .add('Default', () => {
+    let cta, type;
+    const copy = ['Lorem ipsum dolor sit amet', 'Consectetur adipisicing elit'];
+    const types = ['local', 'jump', 'external'];
+    const styles = ['text', 'card', 'feature', 'button'];
+    const style = select('style', styles, styles[0]);
+
+    const target = {
+      external: '_blank',
+      jump: '_self',
+    };
```

Comment:
So I'm afraid there might be a little confusion here. We don't want people passing in this object. The `type` prop we're passing in should decide whether it's `_blank`, smooth scroll, or none.

Reply:
But we should mention somewhere to assign a target value based on type right?

Reply:
Maybe i'm not understanding your question, but they won't assign the target value. We don't want them to assign the target. The component will assign it them. The adopter will pass in `type`, and then we'll use a conditional statement to decide what target value to provide, or if we need to provide one at all.